packages/donate-button-beta/src/components/widget/types/language.ts

```diff
@@ -16,6 +16,8 @@ export type Language = {
 	countrySelection: string;
 	donate: string;
 	donationRedirectNotice: string;
-	nonprofitDescription: string;
 	thanksDonation: string;
+	countryDescription: string;
+	chooseAnAmount: string;
+	enterAnAmount: string;
```

Comment:
can also eventually just derive this type from the actual english i18n values in the future using the `typeof` keyword.